src/locales/en.json

```diff
@@ -171,18 +171,18 @@
       "frontline": "An essential or front-line worker in your area",
       "nonFrontline": "Someone in your area who is NOT an essential or front-line worker",
       "livingAlone": "Lives alone and only grocery shops",
-      "livingWithPartner": "Lives with partner, both people only grocery shop",
+      "livingWithPartner": "Lives with partner, both only grocery shop",
       "closedPod4": "In a closed pod of 4 people",
       "closedPod10": "In a closed pod of 10 people",
       "closedPod20": "In a closed pod of 20 people",
-      "contact1": "Has 1 close contact (incl. roommates) who socializes but does not work. No other work or socializing.",
-      "contact4": "Has 4 close contacts (incl. roommates) who socialize but do not work. No other work or socializing.",
-      "contact10": "Has 10 close contacts (incl. roommates) who socialize but do not work. No other work or socializing.",
+      "contact1": "Has 1 close contact (incl. roommates) who socializes, but does not work. No other work or socializing.",
+      "contact4": "Has 4 close contacts (incl. roommates) who socialize, but does not work. No other work or socializing.",
```

Comment:
Wrong conjugation of "do"

Reply:
Isn't this describing a single person? (contact4)